tests/restart/07-waiting.t

```diff
@@ -16,6 +16,9 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #-------------------------------------------------------------------------------
 # Test restarting a simple suite with a waiting task
+
+# TODO: this test is not very meaningful in SoD
```

Comment:
Agreed, it would be good to strip out some of these tests, though I appreciate that it'll be a long job, perhaps best done later on.

Reply:
Yep.